src/methods/query/getTradePrice.ts

```diff
@@ -24,9 +24,9 @@ export async function getTradePrice(asset1Id: string, asset2Id: string, tradeAmo
                 return;
 
               if (actionType === 'sell') {
-                  amount = new BigNumber(await wasm.get_sell_price(assetsAmounts.asset1, assetsAmounts.asset2, tradeAmount.toString()));
+                  amount = new BigNumber(await wasm.calculate_in_given_out(assetsAmounts.asset1, assetsAmounts.asset2, tradeAmount.toString()));
```

Comment:
The functions should be reversed. When you sell you need to calculate out given in.

Reply:
I used instructions from Martin :) But I've fixed this.